dist/golang/protocol/fees.go

```diff
@@ -395,14 +395,13 @@ func EstimatedTransferResponse(requestTx *wire.MsgTx, dustLimit uint64, feeRate
 				fundingForTokens[i] += dustLimit
 			}
 
-		} else { // Output for BSV to track participant
-			for range asset.AssetSenders {
-				p2PKHOutputCount++
-				fundingForBSV[i] += dustLimit
-			}
-			if len(asset.AssetSenders) == 0 {
+		} else {
+			for _, sender := range asset.AssetSenders {
+				// For the amount that needs to be sent
+				fundingForBSV[masterContractIndex] += sender.Quantity
+				// For extra output that links to the sender
 				p2PKHOutputCount++
```

Comment:
A bitcoin sender doesn't get a "notification" output. There is no settlement balance to post. Only bitcoin receivers get outputs to receive the bitcoin.

Reply:
Looks to me like the smart contract response includes an output of dust to the bitcoin sender. Is this a bug?

Reply:
This smart contract bug has been fixed, and the code modified accordingly.